src/libraries/System.Formats.Tar/tests/TarReader/TarReader.TarEntry.ExtractToFile.Tests.Unix.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Formats.Tar.Tests
+{
+    public partial class TarReader_TarEntry_ExtractToFile_Tests : TarTestsBase
+    {
+        [PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.tvOS)] // https://github.com/dotnet/runtime/issues/68360
+        [Fact]
+        public async Task SpecialFile_Unelevated_Throws_Async()
+        {
+            using TempDirectory root = new TempDirectory();
+            using MemoryStream ms = GetTarMemoryStream(CompressionMethod.Uncompressed, TestTarFormat.ustar, "specialfiles");
+
+            TarReader reader = new TarReader(ms);
+            await using (reader)
+            {
+                string path = Path.Join(root.Path, "output");
+
+                // Block device requires elevation for writing
```

Comment:
do we need to make this test conditional then? or is the `PlatformSpecific` attribute applied above enough?

Reply:
It seems `[PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.tvOS)]` is good enough.